CoreAardvark/ARKLogStore.m

```diff
@@ -128,12 +130,17 @@ - (void)clearLogsWithCompletionHandler:(nullable dispatch_block_t)completionHand
     }
 }
 
+- (void)waitForPendingLogsThenSaveAndWait:(BOOL)wait;
```

Comment:
I'm not sure how much the `wait` parameter gives us here, since we're already waiting for one thing. Maybe just `waitUntilAllLogsConsumedAndArchiveSaved`?